### PR TITLE
TW-936 v10.10 customizable keyboard shortcuts

### PR DESCRIPTION
**Pull request by claireriley**

### CODE REVIEW

File: src/pages/docs/getting-started/settings.md [hunk 1]
```diff
@@ -123,7 +123,18 @@ Select a **Light** or **Dark** theme for Postman. If you're using the Postman de
 
 ## Shortcuts
 
-The **Shortcuts** tab displays all the keyboard shortcuts available in Postman. To turn off keyboard shortcuts, select the **Shortcuts** toggle.
+The **Shortcuts** tab displays all the keyboard shortcuts available in Postman. You can use the default shortcuts, or if you're using the Postman desktop app customize them according to your preferences.
```
Comment (HeatherStenson) [inline src/pages/docs/getting-started/settings.md:126]:
```suggestion
The **Shortcuts** tab displays all the keyboard shortcuts available in Postman. You can use the default shortcuts, or customize them if you're using the Postman desktop app.
```

Comment (HeatherStenson) [reply src/pages/docs/getting-started/settings.md:126]: Maybe nix "according to your preferences" to simplify the sentence?

File: src/pages/docs/getting-started/settings.md [hunk 2]
```diff
@@ -123,7 +123,18 @@ Select a **Light** or **Dark** theme for Postman. If you're using the Postman de
...
+
+To customize a shortcut, select it and then enter your preferred shortcut. Custom shortcuts must meet the following requirents:
+
+* Shortcuts can't overlap with system shortcuts.
+* Shortcuts can't overlap with existing Postman shortcuts.
+* **[macOS only]** Shortcuts must include the (⌃), (⌥), (⇧), or (⌘) key.
+* **[Windows only]** Shortcuts must include the Ctrl, Alt, or Shift key.
```
Comment (HeatherStenson) [inline src/pages/docs/getting-started/settings.md:133]: Per the style guide, it looks like the key names should be bolded - also, I thiiiiink the special chars will also be bolded and shouldn't be in parentheses unless you're saying the name of the key first (so like "press the Command key (⌘)")

**Review by HeatherStenson — APPROVED** [review]
Looks good! 🍊

File: src/pages/docs/getting-started/settings.md (hunk 2 above)
Comment (claireriley) [reply src/pages/docs/getting-started/settings.md:133]: Hmm okay will update. Took these directly from the app so this is how the error will appear to users if they don't include those keys

**Review by mark-anthony-postman — APPROVED** [review]
What Heather said, otherwise LGTM! 😀

File: src/pages/docs/getting-started/settings.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/getting-started/settings.md:133]: Yes to bolding key names like **Ctrl**, **Alt**, or **Shift**

For macOS, I think you could use the parentheses as Heather suggests: 
**Control** (**⌃**), **Option** (**⌥**), **Shift** (**⇧**), or **Command** (**⌘**) key

File: src/pages/docs/getting-started/settings.md [hunk 3]
```diff
@@ -123,7 +123,18 @@ Select a **Light** or **Dark** theme for Postman. If you're using the Postman de
...
+The **Shortcuts** tab displays all the keyboard shortcuts available in Postman. You can use the default shortcuts, or customize them if you're using the Postman desktop app.
+
+To customize a shortcut, select it and then enter your preferred shortcut. Custom shortcuts must meet the following requirents:
+
+* Shortcuts can't overlap with system shortcuts.
+* Shortcuts can't overlap with existing Postman shortcuts.
+* **[macOS only]** Shortcuts must include the **Control** (**⌃**), **Option** (**⌥**), **Shift** (**⇧**), or **Command** (**⌘**) key.
+* **[Windows only]** Shortcuts must include the **Ctrl**, **Alt**, or **Shift** key.
+
+You can revert to the default shortcuts at any time by selecting **Restore Defaults**. To turn off keyboard shortcuts entirely, select the **Keyboard shortcuts** toggle.
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/settings.md:135]:
```suggestion
You can revert to the default shortcuts by selecting **Restore Defaults**. To turn off keyboard shortcuts entirely, select the **Keyboard shortcuts** toggle.
```

**Review by jkonrath-postman — APPROVED** [review]
LGTM.

**Pull request merged**